invisible_cities/cities/dorothea.py

```diff
@@ -133,9 +144,56 @@ def _create_kr_event(self, evt_number, evt_time, S1, S2, Si):
             evt.R    .append((x**2 + y**2)**0.5)
             evt.Phi  .append(np.arctan2(y, x))
 
-            dt  = s2time - evt.S1t[0] if len(evt.S1t) > 0 else -1e3
-            dt *= units.ns / units.mus
+            dt  = evt.S2t[peak_no] - evt.S1t[0]
+            dt  *= units.ns / units.mus
             evt.DT   .append(dt)
             evt.Z    .append(dt * units.mus * self.drift_v)
 
         return evt
+
+    # def _create_kr_event(self, evt_number, evt_time, S1, S2, Si):
```

Comment:
Is there any reason for this to be here? You have written a new version of the function and the old one is commented.